v-next/hardhat-errors/src/descriptors.ts

```diff
@@ -617,18 +617,18 @@ Please ensure that an action is defined for each task.`,
         websiteDescription:
           "The task is empty. Please ensure that tasks have at least one action.",
       },
-      INVALID_ACTION_URL: {
+      INVALID_ACTION_IMPORT: {
         number: 411,
         messageTemplate:
-          'Unable to import the action specified by task "{task}" from the module "{action}"',
-        websiteTitle: "Invalid action URL",
+          'Unable to import the action for task "{task}" because the specified module could not be found.',
```

Comment:
```suggestion
          'Unable to import the action for task "{task}".',
```

And the rest of the fields.

The reason is that the file may be present, but throw.